kalite/i18n/__init__.py

```diff
@@ -351,21 +351,6 @@ def set_default_language(lang_code):
     Settings.set("default_language", lcode_to_ietf(lang_code))
 
 
-def get_langs_with_subtitle(youtube_id):
```

Comment:
All logic for this is now used in the one place where this function was referenced.